pkg/resource/events-logger-secret/grafana-agent-secret.go

```diff
@@ -44,12 +44,17 @@ func GenerateGrafanaAgentSecret(lc loggedcluster.Interface, credentialsSecret *v
 		return v1.Secret{}, errors.WithStack(err)
 	}
 
+	tenant := clusterName
+	if lc.IsCAPI() {
+		tenant = common.DefaultWriteTenant
+	}
+
```

Comment:
It would be best to add a `GetTenant` method in the [logged-cluster](https://github.com/giantswarm/logging-operator/tree/main/pkg/logged-cluster) package to get the the tenant rather than relying on IsCAPI.

Reply:
Done